filter_plugins/exclude.py

```diff
@@ -0,0 +1,21 @@
+from jinja2.runtime import Undefined
```

Comment:
Should add some comments as to why this filter plugin is needed, and for the functions within.

Reply:
Comments have been commentified.